### DISCUSSION TITLE
Load nemesis output files into SolutionUserObject

### DISCUSSION THREAD

**Discussion by zhang-rite** [idaholab/moose#17226] [Q&A General] [answered]
Hi, 

Now another problem related to distributed mesh is how to read the series of nemesis output files (e.g., e.*.*) into a SolutionUserObject for the initial condition in next run? Is there a function for this available? Or for converting nemesis output to an exodus file.

Thank you.

**Comment by GiudGiud** [comment]
@loganharbour can you please answer this or tag someone knowledgeable about nemesis

**Comment by aeslaughter** [comment, marked as answer]
SolutionUserObject doesn't support Nemesis, so to do this correctly that would need to be added. You can use Paraview to convert the Nemesis files to a single Exodus. You can even limit the output to a certain variable. Of course, if the mesh is very large that will probably give you memory problems.
